addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -25,54 +29,208 @@ const SharedPopupWidget = publicWidget.Widget.extend({
         // `contenteditable=true` attribute in edit mode. It will result in a
         // ugly white bar.
         // tl;dr: this is keeping those 2 elements visibility synchronized.
-        'show.bs.modal': '_onModalShow',
-        'hidden.bs.modal': '_onModalHidden',
-    },
+        "_root": {
+            "t-on-show.bs.modal": () => this.popupShown = true,
```

Comment:
feels weird. are you sure that you don't mean "this.popupShown === true", or simply "this.popupShown"?

Reply:
No, I removed a method `onModalShow` that just removed the `d-none` class (while `onModalHidden` added it). Here I set the class through `t-att-class`, and so I update `this.popupShown` on show / on hidden.
Maybe it should just remain in its own method?